js/store.js

```diff
@@ -2356,17 +2345,17 @@ let SearchPageClass = (function(){
         }
 
         let position;
-        if (priceInfo.right) {
+        if (currency.format.right) {
             position = "afterend";
         } else {
             position = "beforebegin";
         }
 
         let notpriceabove_val = document.querySelector("#es_notpriceabove_val");
-        notpriceabove_val.insertAdjacentHTML(position, "<span id='es_notpriceabove_val_currency'>" + priceInfo.symbolFormat.trim() + "</span>");
+        notpriceabove_val.insertAdjacentHTML(position, "<span id='es_notpriceabove_val_currency'>" + currency.format.symbol.trim() + "</span>");
 
         if (SyncedStorage.get("priceabove_value")) {
-            notpriceabove_val.value = new Price(SyncedStorage.get("priceabove_value")).toString().replace(/[^\d,\.]/, '');
+            notpriceabove_val.value = new Price(SyncedStorage.get("priceabove_value"), Currency.customCurrency).toString().replace(/[^\d,\.]/, '');
```

Comment:
This should be `storeCurrency`

Reply:
Thank you, https://github.com/tfedor/AugmentedSteam/pull/122/commits/c64bc6c5ac96ff7579f73906239f5791ecc31e79